pkgs/top-level/all-packages.nix

```diff
@@ -20053,7 +20053,9 @@ with pkgs;
 
   vtable-dumper = callPackage ../development/tools/misc/vtable-dumper { };
 
-  wails = callPackage ../development/tools/wails { };
+  wails = callPackage ../development/tools/wails {
+    stdenv = gccStdenv;
+  };
```

Comment:
Is this really necessary on darwin?

Reply:
Thanks for asking, @wegank. When taking this on I did not want to switch the darwin build from gcc to clang without further investigation.
Yet local tests show that with `stdenv = if stdenv.isLinux then gccStdenv else stdenv;` wails also builds on aarch64-darwin. Although I haven't done any extensive testing with the resulting binary.
Anything in particular that would be worthwhile to test/scrutinize?

/ccc @ianmjones

Reply:
FYI, I tested without `stdenv = gccStdenv;` on x86_64-darwin, and apps would not compile with `wails`, not even the tiny default app created with the `svelte` template.

Reply:
Ah, thanks for the info. Let's leave it as is.